src/Models/Laravel/Attribute.php

```diff
@@ -17,6 +17,7 @@ class Attribute extends BaseModel
     use AttributeMethods;
     use EagerLoadSelectFields;
     use HasFactory;
+    public const URL_ENCODED_KEY = '0x757269';
```

Comment:
Please could we move this into a different class, it feels like the `MetadataService` class might be more appropriate as this model class may not be used if a different database tech stack is used.